scripts/submitSparkPi.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#command should be run like so:
+#  examples/submitSparkPi.sh armada://localhost:30002 ~/incoming/spark
+
+ARMADA_MASTER=$1
+SPARK_ROOT=$2
+ARMADA_SPARK_ROOT=`pwd`
+ARMADA_QUEUE=test
+# Ensure our "test" queue exists on Armada
+armadactl get queues | grep -q -w $ARMADA_QUEUE
+if [ $? -ne 0 ]; then
```

Comment:
Secondly, this can be merged into one line
```suggestion
if armadactl get queue $ARMADA_QUEUE; then
```